src/pkg/packager/actions/actions.go

```diff
@@ -130,7 +130,7 @@ retryCmd:
 		// If no timeout is set, run the command and return or continue retrying.
 		if actionDefaults.MaxTotalSeconds < 1 {
 			spinner.Updatef("Waiting for \"%s\" (no timeout)", cmdEscaped)
-			if err := tryCmd(context.TODO()); err != nil {
+			if err := tryCmd(context.Background()); err != nil {
```

Comment:
Passing in the main `ctx` I think is causing flakes because it's being cancelled?

Reply:
Oh, it needs cancelled.